rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -88,19 +81,13 @@
 
 import static org.hamcrest.Matchers.lessThanOrEqualTo;
 import static org.hamcrest.core.Is.is;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertNotEquals;
-import static org.junit.jupiter.api.Assertions.assertNull;
-import static org.junit.jupiter.api.Assertions.assertThrows;
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
This can lead to a larger JAR file because it may include unused assertions. However this should not have a huge impact because the actual code should be small

Reply:
Agree with @apancorb. Aside from that, the compilation might take longer as well. Anyway, we can discuss how to address this topic for the next PRs

Reply:
As discussed, using wild card to import does not affect the runtime performance at all because the compiler will take care of it and import exactly what is being used.

Besides that, I prefer to import every class individually, makes it easier to read where it is coming from in an environment like this.